packages/ui/src/components/vuestic-components/va-checkbox/VaCheckbox.spec.js

```diff
@@ -25,4 +26,7 @@ describe('VaCheckbox', () => {
   it('is Form Component', () => {
     expect(() => testIsFormComponent(VaCheckbox)).not.toThrow()
   })
+  it('is Selected Component', () => {
```

Comment:
It would make sense to include `testIsFormComponent` (and possibly other mixin checks) in `testIsSelectableComponent`.